doc/exts/docscrape.py

```diff
@@ -1,14 +1,12 @@
 """Extract reference documentation from the NumPy source tree.
 
 """
-from __future__ import print_function
-from __future__ import division
-from __future__ import absolute_import
+from __future__ import absolute_import, division, print_function
 
 import inspect
-import textwrap
-import re
 import pydoc
+import re
+import textwrap
 from StringIO import StringIO
```

Comment:
CR would be nice here, to divide `imports` and `from`

Reply:
Note that PEP 8 also makes [recommendations](https://www.python.org/dev/peps/pep-0008/#imports) about order and grouping of imports (which flake8 doesn't check unfortunately).